build/cross-images/x86_64-unknown-linux-gnu.dockerfile

```diff
@@ -1,9 +1,15 @@
 FROM quickwit/cross-base:x86_64-unknown-linux-gnu
```

Comment:
Two questions:
- What about installing `protoc` in the base image?
- What about installing `protoc` with the package manager: `apt-get install protobuf-compiler`?

Reply:
The `protobuf-compiler` package installs protoc version `3.6.1` this does not seem to support the experimental flag
`Error: Custom { kind: Other, error: "protoc failed: Unknown flag: --experimental_allow_proto3_optional\n" }`
We want a more recent version of protoc

Also, I am now switching to cross base image as they have updated to recent ubuntu versions. I think this can even help solve the rockdb-musl issue.